tests/testthat/test-bm-row-group-size.R

```diff
@@ -12,3 +12,5 @@ test_that("row_group_size benchmark runs", {
     "arrowbench_results"
   )
 })
+
+wipe_results()
```

Comment:
I don't think this should be in some benchmarks but not all. If it's going to affect all, it should be at the top level where one call handles everything.